pipelines/main/platforms/build_linux.soft_fail.arches

```diff
@@ -1,5 +1,5 @@
 # ROOTFS_IMAGE_NAME    TRIPLET                   ARCH           ARCH_ROOTFS    MAKE_FLAGS                               TIMEOUT     ROOTFS_TAG    ROOTFS_HASH
-package_musl           x86_64-linux-musl         x86_64         x86_64         JL_CFLAGS=-Werror,JL_CXXFLAGS=-Werror    .           v6.00         948ca1e496231e4c280c236a3e9bb01c95c2cda5
+package_musl           x86_64-linux-musl         x86_64         x86_64         JL_CFLAGS=-Werror,JL_CXXFLAGS=-Werror    .           v6.00         0566496527284ed8b638273babbb5735b44d2b7e
```

Comment:
Err, maybe I copied something incorrectly, since it seems like this didn't update it?
```
[ Info: Unpacking /tmp/jl_MpK4Yj-download.gz into /cache/julia-buildkite-plugin/depots/8c032ad4-259f-4c1c-9f4f-7d5a0ea47bcd/artifacts/jl_Z7oAHi...
┌ Error: Tree Hash Mismatch!
│   Expected git-tree-sha1:   0566496527284ed8b638273babbb5735b44d2b7e
│   Calculated git-tree-sha1: 948ca1e496231e4c280c236a3e9bb01c95c2cda5
└ @ Pkg.Artifacts /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Pkg/src/Artifacts.jl:369
┌ Error: Unable to download rootfs!
│   treehash = "0566496527284ed8b638273babbb5735b44d2b7e"
│   url = "https://github.com/JuliaCI/rootfs-images/releases/download/v6.00/package_musl.x86_64.tar.gz"
└ @ Main /etc/buildkite-agent/plugins/github-com-staticfloat-sandbox-buildkite-plugin-v1/lib/generate_sandboxed_bash.jl:34
```